README.md

```diff
@@ -64,6 +64,8 @@ Thanks a lot to all the previous [contributors](https://github.com/i18next/i18ne
 - **-l, --locales <list>**: The locales in your applications. Defaults to `en,fr`
 - **--directoryFilter**: Globs of folders to filter
 - **--fileFilter**: Globs of files to filter
+- **--keep-removed**: Prevent keys no longer found from being removed
+- **--no-old**: Avoid saving the _old files
```

Comment:
I think the README needs to be updated. `no-old` should be `writeOld`. Maybe even: `**--write-old false**: Avoid saving the _old files`.